Source/RhetosCli/Program.cs

```diff
@@ -17,18 +17,149 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+using DeployPackages;
+using Rhetos;
+using Rhetos.Logging;
+using Rhetos.Utilities;
 using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.Reflection;
 
-namespace RhetosCli
+namespace Rhetos
 {
-    class Program
+    public static class Program
     {
-        static void Main(string[] args)
+        private readonly static Dictionary<string, string> _validArguments = new Dictionary<string, string>()
         {
+            { "/StartPaused", "Use for debugging with Visual Studio (Attach to Process)." },

```

Comment:
Remove the dp.exe cli argumets. many of them are not needed in rhetos.exe, we can add them later when we develop the new infrastructure.